src/components/Todos/TodosFooter.tsx

```diff
@@ -0,0 +1,74 @@
+import { FC, memo, useContext } from 'react';
+import { TodosContext } from './TodosContext';
+
+export const TodosFooter: FC = memo(() => {
+  const {
+    todos,
+    setTodos,
+    filterType,
+    setFilterType,
+  } = useContext(TodosContext);
+
+  const total = todos.length;
+  const completed = todos.filter(t => t.completed).length;
+  const active = total - completed;
+
+  return (
+    <div className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        {`${active} items left`}
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
+          className={`filter__link ${
+            filterType === 'all' ? 'selected'
+              : ''} ${
+            !total ? 'is-disabled' : ''}`}
```

Comment:
i still feel weird about ternaries in classnames 🙂 
frankly they're hard to comprehend